### PR TITLE
Check if group member exist in the group before creating it

### PR DESCRIPTION
**Pull request by dkorn**

### CODE REVIEW

File: library/hawkular_alerts_member.py [hunk 1]
```diff
@@ -112,6 +112,19 @@ def group_trigger_exist(self, group_id):
...
+    def group_member_exist(self, group_id, name):
+        """
+        Searches the member name in the group trigger's members
+
+        Returns:
+            True if the member already exist, False otherwise
+        """
+        try:
+            group_members = self.client.get_group_members(group_id)
+            return next((True for gm in group_members if gm.name == name), False)
```
Comment (cben) [inline library/hawkular_alerts_member.py]: I'd move this line outside the try..except.

**Review by cben — APPROVED** [review]
LGTM.

I suspect you could drop `group_trigger_exist()` and rely on `group_member_exist()` failing if the group doesn't exist.  OTOH checking both is nicely explicit, and might give clearer error message.  Your call.

File: library/hawkular_alerts_member.py (hunk 1 above)
Comment (dkorn) [reply library/hawkular_alerts_member.py]: Done, thanks!

**Comment by dkorn** [comment]
@enoodle please review

**Pull request merged**